samples/vision_edge/edge_container_predict/automl_vision_edge_container_predict.py

```diff
@@ -41,9 +68,11 @@ def container_predict(image_file_path, image_key, port_number=8501):
     Returns:
         The response of the prediction request.
     """
-
-    with io.open(image_file_path, 'rb') as image_file:
-        encoded_image = base64.b64encode(image_file.read()).decode('utf-8')
+    # AutoML edge models will preprocess the input image.
```

Comment:
Check branding guidelines: "Edge" should be capitalized and should use "Vision" before it.

Reply:
done